components/bitbadges/actions/complete-claim.mjs

```
@@ -0,0 +1,87 @@
+    }
+    
+    // Sanitize and validate each part
+    const [claimId, passwordPluginId, password] = details.map(part => part.trim());
+    if (!/^[a-zA-Z0-9-_]+$/.test(claimId)) {
+      throw new Error("Invalid claim ID format");
+    }
+    if (!/^[a-zA-Z0-9-_]+$/.test(passwordPluginId)) {
+      throw new Error("Invalid password plugin ID format");
+    }
+
+    const claimId = details[0];
```

Biome (1.9.4)

[error] Shouldn't redeclare 'claimId'. Consider to delete it or rename it.

'claimId' is defined here:


(lint/suspicious/noRedeclare)